skimage/restoration/_denoise.py

```diff
@@ -286,8 +290,23 @@ def denoise_tv_bregman(image, weight, max_iter=100, eps=1e-3, isotropic=True):
     shape_ext = (rows + 2, cols + 2, dims)
 
     out = np.zeros(shape_ext, image.dtype)
-    _denoise_tv_bregman(image, image.dtype.type(weight), max_iter, eps,
-                        isotropic, out)
+
+    if multichannel:
+        for c in range(image.shape[-1]):
+            channel_in = np.ascontiguousarray(np.atleast_3d(image[..., c]))
```

Comment:
```suggestion
           # the algorithm below expects 3 channels to always be present.
           # slicing the array in this fashion preserves the 3rd dimension for us
            channel_in = np.ascontiguousarray(image[..., c:c+1])
```

Reply:
I don't particularly like `atleast_3d` since I don't know where the dimension is added from its name.

Reply:
totally fine with me ;)

Reply:
ultimately, the comment above is more important than anything else ;)